datafiles/tests/test_converters.py

```diff
@@ -161,6 +177,12 @@ def when_immutable(expect, converter, data, value):
                 (MyDict, None, {}),
                 (MyDict, {}, {}),
                 (MyDict, {'a': 1}, {'a': 1}),
+                (IntegerSet, set(), set()),
+                (IntegerSet, '1, 2.3', {1, 2}),
```

Comment:
I think it would be worth adding another case showing that `'1, 1.1, 1.9, 2, 3.1'` results in `{1, 2, 3}`, demonstrating the set behavior.